.github/workflows/deploy.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Retrieve release distributions
         uses: actions/download-artifact@v4
         with:
-          name: release-dists
+          name: release-dist
```

Comment:
could you make it `release-dists` to match the docs https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-pypi